### DISCUSSION TITLE
Trying out CTK 5.0.2: RadioButtons are weird

### DISCUSSION THREAD

**Discussion by MegMM** [TomSchimansky/CustomTkinter#793] [Q&A] [answered]
Hello,
First, just wanted to thank you, Tom, for providing such an incredibly useful library to the Python community!!

After upgrading to 5.0.2, and making a few corrections that the newer version didn't like, I got my GUI running but it looks a lot different. Especially the RadioButtons. I tried changing the width and height and they're still rather massive. With CTK 4  `height=12, width=12` just about matched the height of the text font. With CTK 5 they're much larger. I tried shrinking them down to 6 and then 2, and noticed hardly any difference. The circles are still much larger than my text font. _Forgot to add I'm using `ctk.set_appearance_mode('dark)
ctk.set_default_color_theme('blue)` if that makes a difference._ Why is this?
Thanks!
-Meghan

**Comment by MegMM** [comment, marked as answer]
I've found a couple of clarifications in the docs: using `radiobutton_height, radiobutton_width` has fixed the buttons. It's just the widget acting _differently_ than in CTK 4. 

Switching my old code in CTK 4:
````
    width=12, height=12, 
    border=2,
````
to this new code for CTK 5 produces the same results as before
```
    radiobutton_width=12, radiobutton_height=12, 
```

However, I'm not sure about `border_width_checked=3, border_width_unchecked=2` because I'm getting a `['border_width_checked', 'border_width_unchecked'] are not supported arguments` error:
```
        radio_3 = ctk.CTkRadioButton(master=save_frame, 
                                     radiobutton_width=12, radiobutton_height=12, 
                                     border_width_checked=3, border_width_unchecked=2,                                   
                                     text='Save files', 
                                     value=1, variable=self.save_option_var,
                                     command=lambda: self.save_on_click(radio_3, save_frame))
```
```
File "C:\Users\megha\AppData\Local\Programs\Python\Python311\Lib\site-packages\customtkinter\windows\widgets\ctk_checkbox.py", line 47, in __init__
    super().__init__(master=master, bg_color=bg_color, width=width, height=height, **kwargs)
  File "C:\Users\megha\AppData\Local\Programs\Python\Python311\Lib\site-packages\customtkinter\windows\widgets\core_widget_classes\ctk_base_class.py", line 48, in __init__
    check_kwargs_empty(kwargs, raise_error=True)
  File "C:\Users\megha\AppData\Local\Programs\Python\Python311\Lib\site-packages\customtkinter\windows\widgets\utility\utility_functions.py", line 18, in check_kwargs_empty
    raise ValueError(f"{list(kwargs_dict.keys())} are not supported arguments. Look at the documentation for supported arguments.")
ValueError: ['border_width_checked', 'border_width_unchecked'] are not supported arguments. Look at the documentation for supported arguments.
```

**Reply by TomSchimansky** [reply]
Yes width and height in version 4 described the dimensions of the button. In version 5 width and height now describe the dimensions of the whole widget with text like for every other widget. And the button size is set be the radiobutton_width and radiobutton_height attributes. The same happened for the switch and the checkbox.

**Reply by MegMM** [reply]
Thank you! The `border_width_checked/unchecked` options show up in the GUI but but when I press the button it gives me that error.
